### PR TITLE
Uncomments structure for defaults

### PR DESCRIPTION
**Pull request by brianjhanson**
This uncomments our structure examples and adds them as defaults. This way you can use the mixins out of the box (even though they won't be what you want). It also means you can define your own variables _after_ these are declared. 

In other words you can do:
```
@import 'one-sass-toolkit/spacing'
@import 'your-variables'
```

### CODE REVIEW

**Comment by cmalven** [comment]
If we're going to do this, I think we should remove the Futura font stack, or at least replace it with something that will work by default on every machine. Seems weird to have defaults that don't work.

File: _type-styles.scss [hunk 1]
```diff
@@ -7,44 +7,44 @@
...
-      sizes: (
-        default: 16,
-        medium: 24
-      )
+  ),
+
+  body: (
+    stack: helvetica,
+    line-height: 1.4,
+    text-transform: uppercase,
```
Comment (cmalven) [inline _type-styles.scss]: Also, weird to have the default `text-transform` for body be `uppercase`. This was probably an accident.

File: _type-styles.scss [hunk 2]
```diff
@@ -7,44 +7,44 @@
...
+    letter-spacing: 1.2,
+    font-smoothing: true,
```
Comment (cmalven) [inline _type-styles.scss]: Let's remove the `letter-spacing` and `font-smoothing` defaults for the `body` style.

**Comment by brianjhanson** [comment]
Updated.

**Comment by brianjhanson** [comment]
Sorry, didn't look at the rest of it that closely. 

I'd also like to push an update that makes more of those arguments optional, but that's for another day.

**Comment by cmalven** [comment]
One last thing, let's bump the package version so we can publish after merge.

**Comment by brianjhanson** [comment]
@cmalven bumped version number.

P.S. Can you add me to the repo?

**Comment by cmalven** [comment]
@brianjhanson added you

Did you mean to close this PR?

**Comment by brianjhanson** [comment]
Yeah, I rebased off master without thinking and lost the changes I just did. Didn't want you to swoop in and merge prematurely. Un momento

**Comment by brianjhanson** [comment]
All good now. Sorry about that.

File: _type-styles.scss [hunk 3]
```diff
@@ -24,24 +24,23 @@ $font-stacks: (
...
 
   body: (
-    stack: helvetica,
-    line-height: 1.4,
-    text-transform: uppercase,
-    letter-spacing: 1.2,
-    font-smoothing: true,
+    stack: serif,
+    line-height: 1.6,
+    text-transform: normal,
```
Comment (cmalven) [inline _type-styles.scss:40]: `text-transform` and `letter-spacing` actually _are_ optional, so we could remove them here if we wanted. I think we should include them somewhere (even if just in the README) to make it clear that you can modify them, but you can remove them here if you want.

Comment (cmalven) [reply _type-styles.scss:40]: (or let me know if you're not interested in that and I'll merge)

**Comment by brianjhanson** [comment]
I am interested in that, but I'm thinking it should be a more robust documentation of how the function works. Not sure if we should put that in the README or in the code itself. Also, trying to figure out exactly what form it should take.

**Comment by cmalven** [comment]
It should go either in the README or in the README + code, I'd think. But that can be a separate release.

**Comment by brianjhanson** [comment]
I'm good with that.

**Comment by cmalven** [comment]
Please do an npm publish when you get the chance.

**Comment by brianjhanson** [comment]
Published.

**Pull request merged**